src/osvr/Common/CommonComponent.cpp

```diff
@@ -35,8 +35,24 @@ namespace osvr {
 namespace common {
     namespace messages {
 
+        // These messages all must match the ones in VRPN exactly.
+        /// @todo add tests to verify that these message identifiers match VRPN
+        /// string constants.
+
```

Comment:
well, it's probably more likely to actually get done if it blocks the merge, if I'm being honest about it.